components/brave_wallet_ui/options/asset-options.ts

```diff
@@ -23,6 +23,8 @@ export const getNetworkLogo = (chainId: string, symbol: string): string => {
     return 'chrome://erc-token-images/celo.png'
   if (chainId === BraveWallet.ARBITRUM_MAINNET_CHAIN_ID)
     return 'chrome://erc-token-images/arb.png'
+  if (chainId === BraveWallet.NEON_EVM_MAINNET_CHAIN_ID)
+    return 'chrome://erc-token-images/neon.png'
   if (chainId === AllNetworksOption.chainId)
     return AllNetworksOption.iconUrls[0]
 
```

Comment:
I see some svgs are removed, do they relate somehow to these pngs?
Should we cleanup something there?

Reply:
The svgs were not used anywhere. All token logos should be used via `chrome://erc-token-images/*.png`. Neon logo was added here: https://github.com/brave/token-lists/pull/75